local-data-platform/local_data_platform/__init__.py

```diff
@@ -39,7 +39,7 @@ class Table(Base):
 
     def __init__(self, name: str, path: Path = os.getcwd()):
         self.name = name
-        self.path = path
+        self.path = os.getcwd()+path
```

Comment:
why you have taken os.getcwd() twice, once as argument and other time concatenating ?
pls check